app/services/api/v1x2/catalog/evaluate_order_process.rb

```diff
@@ -0,0 +1,68 @@
+module Api
+  module V1x2
+    module Catalog
+      class EvaluateOrderProcess
+        def initialize(order)
+          @order = order
+        end
+
+        def process
+          applicable_order_item = @order.order_items.first
+
+          relevant_order_processes = find_relevant_order_processes
+
+          # TODO: Put the order processes in order by their sequence number before
+          # doing this logic
+          @before_sequence_number = 1
+          @after_sequence_number = (relevant_order_processes.length * 2) + 1
```

Comment:
why length * 2?

Reply:
Because if there are multiple order processes, they will have both a before and an after. So for example, if there's 3 order processes, the first before should be 1, and the first after should be 7, then 2 and 6, then 3 and 5, and the applicable order item is right in the middle at 4.

So, we want to start at the highest possible and go backwards from there, and since each order process has a before and an after, we have to do `length * 2`.

I'll put this in a private method with a better name, though.